src/ludic.rs

```diff
@@ -0,0 +1,98 @@
+// https://rosettacode.org/wiki/Ludic_numbers
```

Comment:
Due to @euclio 's `coverage` code this needs to be:

```rust
// https://rosettacode.org/wiki/Ludic_numbers
```

This makes the build succeed. :)